css-flexbox/css-flexbox/css-flexbox-quiz-notes.md

```diff
@@ -0,0 +1,45 @@
+# css-flexbox-quiz-notes
+
+## Quiz Questions
+
+After completing this exercise, you should be able to discuss or answer the following questions:
+
+- What is the default `flex-direction` of a `flex` container?
+  The default 'flex-direction' of a 'flex' container is left to right.
```

Comment:
Yes, but it's asking for the specific default value of `flex-direction`. `column`, `row`, `column-reverse`, etc